pkg/db/db.go

```diff
@@ -883,6 +883,33 @@ func (h *DBHandler) DBSelectReleasesByAppOrderedByEslVersion(ctx context.Context
 	return h.processReleaseRows(ctx, err, rows, ignorePrepublishes, false)
 }
 
+func (h *DBHandler) DBSelectLastReleasesByApp(ctx context.Context, tx *sql.Tx, app string, deleted bool, ignorePrepublishes bool) (*DBReleaseWithMetaData, error) {
+	span, ctx := tracer.StartSpanFromContext(ctx, "DBSelectReleasesByApp")
+	defer span.Finish()
+	selectQuery := h.AdaptQuery(fmt.Sprintf(
+		"SELECT eslVersion, created, appName, metadata, releaseVersion, deleted, environments " +
+			" FROM releases " +
+			" WHERE appName=? AND deleted=?" +
+			" ORDER BY eslVersion DESC, releaseVersion DESC" +
+			" LIMIT 1;"))
```

Comment:
If we have two rows like this:
```
eslVersion: 3 releaseVersion: 2
eslVersion: 2 releaseVersion: 3
```
This query will return the releaseVersion 2 but it should return the 3.
We should use joins for this query, like the one [here](https://github.com/freiheit-com/kuberpult/blob/73f2cfdce8e8d17a891244576c0a54bda39ee68a/pkg/db/db.go#L826)